Riot.xcodeproj/xcshareddata/xcschemes/Riot.xcscheme

```diff
@@ -1,11 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think maybe we don't want to merge these xcscheme changes.

Reply:
I have no idea how this got pushed, must have been a mistake, will revert this file.

Reply:
reverted